### PR TITLE
Feat : Add user review section in all portfolio pages

### PR DESCRIPTION
**Pull request by cp-jagruti-a**
**Tested on:**
- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [ ] Laptop / Macbook  (chrome, firefox, safari)
- [x] iPhone(safari)
- [x] Available android devices  (chrome, firefox)
- [x] iPad(safari)
- [x] Tablet (chrome, firefox)

**Page speed score**

Mobile:
Screenshot from 2023-02-20 13-12-57

Desktop:
Screenshot from 2023-02-20 13-14-00

### CODE REVIEW

File: vue-frontend/src/portfolio-json/togness-data.js [hunk 1]
```diff
@@ -260,6 +260,75 @@ export default {
       title: "Justly",
       url: "justly",
       event: "tap_next_project_justly",
+      review: {
+        ref: "justlyreview",
```
Comment (cp-sumi-k) [inline vue-frontend/src/portfolio-json/togness-data.js:264]: It's togness

File: vue-frontend/src/components/portfolio/FooterSection.vue [hunk 2]
```diff
@@ -12,9 +12,133 @@
       />
     </aspect-ratio>
   </section>
-
   <section
-    class="v2-header-2-text tw-font-bold tw-text-center tw-bg-white tw-py-20 sm:tw-py-40 lg:tw-py-80"
+    class="tw-bg-white tw-pt-36 md:tw-pb-0 lg:tw-pt-50"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/portfolio/FooterSection.vue:16]: Let's create separate user review component

Comment (cp-sumi-k) [inline vue-frontend/src/components/portfolio/FooterSection.vue:16]: Let's keep same properties together i.e `pt `. Also reduce bottom space of div

File: vue-frontend/src/components/portfolio/FooterSection.vue [hunk 3]
```diff
@@ -12,9 +12,133 @@
...
+    :ref="response.review ? response.review.ref : ''"
+  >
+    <div class="header-text canopas-gradient-text tw-text-center">
+      Listen what real app users <br />
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/portfolio/FooterSection.vue:20]: Let's make header in black like other headers in portfolio

File: vue-frontend/src/components/portfolio/FooterSection.vue [hunk 4]
```diff
@@ -12,9 +12,133 @@
...
+      are saying...
+    </div>
+    <div
+      class="tw-mt-[32px] tw-text-left tw-overflow-x-hidden tw-overflow-y-auto tw-scrollbar-0"
+      ref="horizontalGrid"
+    >
+      <div
+        class="tw-animate-gridAnimationReverse hover:tw-animate-gridAnimationReverse"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/portfolio/FooterSection.vue:28]: hover and reverse aniamtions are not working

File: vue-frontend/src/components/portfolio/FooterSection.vue [hunk 5]
```diff
@@ -25,15 +149,21 @@
   </section>
 </template>
 
-<script>
+<script type="module">
 import AspectRatio from "@/components/utils/AspectRatio.vue";
+import { analyticsEvent } from "@/utils.js";
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/portfolio/FooterSection.vue:154]: Merge master properly. We have not this method now

File: vue-frontend/src/components/portfolio/FooterSection.vue [hunk 6]
```diff
@@ -12,9 +12,133 @@
...
+        <div
+          class="tw-animate-gridAnimation hover:tw-animate-gridAnimationPaused"
+        >
+          <div
+            v-if="usersReviewsGrid1"
+            class="tw-flex tw-mb-[16px] tw-ml-[160px] sm:tw-ml-[240px] lg:tw-ml-[264px] tw-w-full"
+            ref="horizontalGrid1"
+          >
+            <div
+              v-for="userReview in usersReviewsGrid1"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/portfolio/FooterSection.vue:40]: You can use for loop for appending grid 3 times, instead of adding them manually

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/portfolio/FooterSection.vue [hunk 7]
```diff
@@ -12,9 +12,9 @@
       />
     </aspect-ratio>
   </section>
-
+  <section><UserReviewSection v-bind:json="response.review" /></section>
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/portfolio/FooterSection.vue:15]: This should be in portfolio's main page, check how other components are added

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/portfolio/FooterSection.vue (hunk 7 above)
Comment (cp-jagruti-a) [reply vue-frontend/src/components/portfolio/FooterSection.vue:15]: i have seen other components but we have to that footer is one component and we have to add it in between it that's why added like this

**Review by cp-priyanka-g — DISMISSED** [review]

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**